cmake/Findfirebase_cpp_sdk.cmake

```diff
@@ -38,6 +38,30 @@ if(APPLE)
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
   set(FIREBASE_CPP_LIB_DIR ${FIREBASE_CPP_SDK_DIR}/libs/linux/${CMAKE_SYSTEM_PROCESSOR})
 
+elseif(MSVC)
+  set(MSVC_VS_VERSION VS2015)
+
+  # Windows runtime mode, either MD or MT depending on whether you are using
+  # /MD or /MT. For more information see:
+  # https://msdn.microsoft.com/en-us/library/2kzt1wy3.aspx
+  set(MSVC_RUNTIME_MODE MD)
+
+  if(${CMAKE_CL_64})
+    set(MSVC_CPU x64)
+  else()
+    set(MSVC_CPU x86)
+  endif()
+
+  if(CMAKE_BUILD_TYPE EQUAL Release)
+    set(MSVC_CONFIG Release)
+  else()
+    set(MSVC_CONFIG Debug)
+  endif()
+
+  set(
+    FIREBASE_CPP_LIB_DIR
+    ${FIREBASE_CPP_SDK_DIR}/libs/windows/${MSVC_VS_VERSION}/${MSVC_RUNTIME_MODE}/${MSVC_CPU}/${MSVC_CONFIG}
+  )
```

Comment:
You could add an else clause that would error out saying something about an unsupported platform. (Without that, the error message will be rather obtuse.)

Optional.

Reply:
Done.

Even without this change the failure you would have gotten is this:
```
CMake Error at /Users/mcg/homebrew/Cellar/cmake/3.12.3/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find firebase_cpp_sdk (missing: FIREBASE_CPP_APP_LIBRARY)
```

Now it's this:
```
Unknown platform for the firebase_cpp_sdk.

  CMAKE_SYSTEM_NAME = Darwin
  CMAKE_SYSTEM_PROCESSOR = x86_64

CMake Error at /Users/mcg/homebrew/Cellar/cmake/3.12.3/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
```